packages/components/src/styles/components/time.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TIME COMPONENT
+//
+
+.hds-time-range {
+  display: flex; // remove excess space between time range elements
```

Comment:
Is there any use case where these times would need to be used inline? Like in a paragraph? If so, adding `display: flex` might break support for that by causing the content to go to a new line. 

With `display: flex`
<img width="615" alt="Screenshot 2024-11-15 at 4 36 50 PM" src="https://github.com/user-attachments/assets/a8180dba-dc57-426a-ae9e-2ee8f81ccfcb">
Without `display: flex`
<img width="614" alt="Screenshot 2024-11-15 at 4 36 56 PM" src="https://github.com/user-attachments/assets/f4099ac9-2b44-4651-ad94-ece0eb1029ad">

Reply:
Hi yes I was testing these mainly together with the tooltip button enabled so missed that. Will update and add more Showcase examples as well.

Reply:
Fixed

Reply:
I added examples of the Time components being used within strings of text to the Showcase btw.